be/src/exprs/vectorized/math_functions.cpp

```diff
@@ -153,6 +153,13 @@ DEFINE_UNARY_FN_WITH_IMPL(log2Impl, v) {
 
 DEFINE_MATH_UNARY_WITH_OUTPUT_NAN_CHECK_FN(log2, TYPE_DOUBLE, TYPE_DOUBLE);
 
+// square
+DEFINE_UNARY_FN_WITH_IMPL(squareImpl, v) {
+    return (double)std::pow(v, 2.0);
```

Comment:
```suggestion
    return v * v;
```

Reply:
OK，i will modify to v*v to improve performance